django_rq/dashboard/cli.py

```
@@ -0,0 +1,281 @@
+        print()
+        print("✓ Superuser created successfully!")
+        print()
+    except KeyboardInterrupt:
+        print("\n\nSuperuser creation cancelled.")
+        print("You need a superuser account to access the dashboard.")
+        sys.exit(1)
+    except CommandError as e:
+        print(f"\n✗ Error creating superuser: {e}")
+        print("\nPlease run the command again to retry.")
+        sys.exit(1)
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)